Formula/aldroid.rb

```diff
@@ -0,0 +1,12 @@
+class Aldroid < Formula
+    version "0.1.0"
+
+    desc "This tool transforms Android APKs to make them debuggable and MITMable/Charlesable"
+    homepage "https://github.com/AppLovin/homebrew-Mobile-Tools"
+    url "https://github.com/AppLovin/homebrew-Mobile-Tools/archive/v#{version}.tar.gz"
+
+    def install
+        bin.install "aldroid"
+        bin.install "aldroid-running-app"
+    end
```

Comment:
Do we need a major, minor, and patch version for a tool? Unless you have a specific reason for the patch version, I'm thinking we only need major and minor. Also is the URL for downloading missing the tool name in it?
```suggestion
    # Description
    desc "This tool transforms Android APKs to make them debuggable and MITMable/Charlesable"

    # Version
    version "1.0"

    # Homepage
    homepage "https://github.com/AppLovin/homebrew-Mobile-Tools"
    
    # URL for downloading aldroid
    url "https://github.com/AppLovin/homebrew-Mobile-Tools/archive/v#{version}.tar.gz"

    # Commands
    def install
        bin.install "aldroid"
        bin.install "aldroid-running-app"
    end
```

Reply:
I think it's fine to follow [software semantic versioning rules](https://semver.org/).